core-management/core-management-subsystem/src/main/java/org/wildfly/extension/core/management/CoreManagementRootResourceDefinition.java

```diff
@@ -38,7 +54,40 @@ public Collection<AttributeDefinition> getAttributes() {
     @Override
     protected List<? extends PersistentResourceDefinition> getChildren() {
         return Arrays.asList(ConfigurationChangeResourceDefinition.INSTANCE,
-                new ProcessStateListenerResourceDefinition()
+                new ProcessStateListenerResourceDefinition(),
+                UnstableApiAnnotationResourceDefinition.INSTANCE
         );
     }
+
+    private static class CoreManagementAddHandler extends AbstractBoottimeAddStepHandler {
+
+        @Override
+        protected void performBoottime(OperationContext context, ModelNode operation, Resource resource) throws OperationFailedException {
+
+            if (context.isNormalServer()) {
```

Comment:
Should these DUPs be added conditionally? i.e. if the stability level of the server enables the use of the UnstableApiAnnotationResourceDefinition?  or perhaps more generally, only if the child resource was defined (which presumes the previous condition)?
e.g.
```java
Resource unstableApiResource = resource.getChild(UnstableApiAnnotationResourceDefinition.PATH);
if (unstableApiResource != null) {
   // Read UnstableApiAnnotationLevel from child resource model
   // Install service
   // Add step that adds DUPs
}
```